### ISSUE TITLE
Add backend support for provisioning circumvention tool test configurations to probes

### ISSUE THREAD

**Issue by hellais** [ooni/orchestra#68]
This involved:
* [x] Adding support for giving out psiphon configuration: https://github.com/ooni/orchestra/issues/78
* [x] Add support for giving out addresses for TCP Connect: https://github.com/ooni/orchestra/issues/51
* [x] Add support for giving out addresses for tor test: https://github.com/ooni/orchestra/issues/82

This will be adding some API endpoint to: https://github.com/ooni/orchestra/blob/master/orchestrate/orchestrate/api/v1/api.go and serving all the configs for the various tools.

There will also be some batch process run on the pipeline (similar to sync-test-lists) which will update the various test vectors periodically based on some canonical source of truth.

**Comment by hellais** [comment]
We need to come up with a solution to this so that we can provision to probes configuration information, like psiphon config, in such a way that untrustworthy actors don't abuse this configuration information.

We should have a dedicated meeting with  @FedericoCeratto & @bassosimone to discuss this.

**Comment by bassosimone** [comment]
We came up with a solution for that. We will serve this information using authenticated request. What's more, we let this issue slip through but we actually already completed 50% of it in #72 

What now remains to be done in this cycle is to give out bridge information. Because working on this issue is instrumental to https://github.com/ooni/probe-engine/issues/2, I am going to move this issue back into the cycle backlog. Because 50% of the work has been done by now, I am going to update the effort estimation for this issue. Because it's clear we want to provide more information than just the addresses for TCP connect, I am going to modify the original description for this issue.
